jenkins-ci/Jenkinsfile

```diff
@@ -0,0 +1,57 @@
+pipeline {
+    agent any
+    stages {
+        stage('all-in-one: clean environment') { 
+            steps { 
+                sh 'oc get all'
```

Comment:
As it is right now is fine but delete might not agnostic to the created objects. 

I think you can create a namespace just for tests.